extensions/whox.md

```diff
@@ -77,6 +77,9 @@ MUST be used:
 - Any syntactically correct placeholder can be used for `[oplevel]` when the
   server doesn't support op levels, for instance `n/a`.
 
+Servers MAY return multiple or all applicable prefixes in the `[flags]` (`f`)
+field, regardless of whether [`multi-prefix`](multi-prefix.html) is enabled.
```

Comment:
this should link to `multi-prefix.md` and the site generator will rewrite it

Reply:
Hmm, this would be the only `.md` link in the repository. I based this link on what I found in other documents, which all use `.html`, e.g. in `extensions/echo-message.md` or `extensions/extended-monitor.md`.